sims/verilator/Makefile

```diff
@@ -1,112 +1,175 @@
-#########################################################################################
+#============================================================================
```

Comment:
A couple of nits:
- I think the "verilator makefile" should be above the "Extra make variables" section.
- Can we switch back to the `###########` style separators since they are the consistently used in the other makefiles? I think we should be consistent and try to match the style of the current makefiles (unless `===` is objectively better).

Reply:
i have already reverted back to ### by popular demand